README.md

```diff
@@ -27,3 +28,7 @@ about the current recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+Answer:
+- At the beginning of every Nth recursion there is an array of N+1 integers that includes at minimum 0 in the first position
```

Comment:
But what integers specifically? It's not enough to just have any integers for the Fibonacci numbers to be correct.